login/service_blackbox_test.go

```diff
@@ -218,15 +164,19 @@ func (s *serviceBlackBoxTest) unapprovedUserRedirected() (*string, error) {
 	}
 
 	claims := make(map[string]interface{})
-	claims["approved"] = false
-	accessTokenStr, err := testtoken.GenerateAccessTokenWithClaims(claims)
-	require.Nil(s.T(), err)
-
-	refreshTokenStr, err := testtoken.GenerateRefreshTokenWithClaims(claims)
-	require.Nil(s.T(), err)
+	claims["sub"] = uuid.NewV4().String()
+	accessToken, err := testtoken.GenerateAccessTokenWithClaims(claims)
+	if err != nil {
```

Comment:
please do not panic here, it will kill the test suite. Instead, just use `require.NoError(s.T(), err)`